ouroboros-consensus-byron-test/test/Test/Consensus/Byron/LedgerTables.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Consensus.Byron.LedgerTables (tests) where
+
+import           Ouroboros.Consensus.Byron.Ledger
+
+import           Test.Consensus.Byron.Generators ()
+import           Test.LedgerTables
+
+import           Test.Tasty
+import           Test.Tasty.QuickCheck
+
+tests :: TestTree
+tests = testGroup "LedgerTables"
+  [ testProperty "Stowable laws" (prop_stowable_laws @ByronBlock)
+  , testProperty "TableStuff laws" (prop_tablestuff_laws @ByronBlock)
```

Comment:
Nothing we can do now, but I wonder if we should have renamed table stuff earlier :grimacing:

Reply:
`HasTables`/`HasLedgerTables` :upside_down_face: